bindings/nodejs/lib/types/models/info/node-info-metrics.ts

```diff
@@ -6,15 +6,15 @@
  */
 export interface INodeInfoMetrics {
     /**
-     * Blocks per second.
+     * The current rate of new blocks per second.
      */
-    blocksPerSecond: number;
+    blocksPerSecond: string;
     /**
-     * Referenced blocks per second.
+     * The current rate of confirmed blocks per second.
      */
-    referencedBlocksPerSecond: number;
+    confirmedBlocksPerSecond: string;
     /**
-     * The rate at which rates are being referenced.
+     * The ratio of confirmed blocks in relation to new blocks of the last confirmed slot.
```

Comment:
```suggestion
     * The ratio between confirmed blocks and new blocks of the last confirmed slot.
```

Reply:
but anything is better than the previous doc :smile:

Reply:
Not sure what it's worth but of/to seems preferred https://ell.stackexchange.com/questions/4279/when-describing-a-ratio-should-between-or-of-be-used